extension/src/background/helpers/dataStorage.ts

```diff
@@ -34,3 +34,11 @@ export const migrateLocalStorageToBrowserStorage = async () => {
 
   await dataStorage.setItem(storage);
 };
+
+// TODO - temporary wrapper around localStorage until we replace
+// localStorage all together
+export const freighterLocalStorage = {
+  getItem: (keyId: string) => localStorage.getItem(keyId),
+  setItem: (keyId: string, value: string) => localStorage.setItem(keyId, value),
```

Comment:
Could you also please make this method store in localStorage as well as in `dataStorage` above?

For ex:
```
  setItem: (keyId: string, value: string) => { localStorage.setItem(keyId, JSON.stringifu(value)); dataStorage.setItem({ [keyId]: value }) },

```